dev_guide/deployments.adoc

```diff
@@ -165,27 +209,78 @@ deployment version. The restarted deployment will have the same configuration
 it had when it failed.
 
 == Rolling Back a Deployment
+Use the `oc rollback` command to revert part of an application back to a
+previous deployment:
+
+----
+$ oc rollback <deployment> [<options>]
+----
+
+Rollbacks revert an application back to a previous deployment; including
+environment variable and volumes. Therefore, consider the following when
+deciding whether a rollback is viable or not:
+
+- If security credentials have been recently updated, the previous deployment
+may not have the correct values.
+- If the previous deployment used a custom strategy which is no longer available
+or usable, the deployment may not be deployed correctly.
+
+During a rollback, only the configuration of pods and containers is changed by
+default, while the scaling or trigger settings remain unchanged.
+
+The `-d` or `--dry run` option shows the configuration of the updated deployment
+in an easy to read format without actually executing the rollback. This allows
+you to inspect the output before actually proceeding with the rollback.
+
+.Rollback CLI Configuration Options
+[cols="4,8",options="header"]
+|===
+
+|Option |Description
+
+.^|`--change-triggers`
+|Include the previous deployment's triggers in the rollback.
+
+.^|`--change-strategy`
+|Include the previous deployment's strategy in the rollback.
+
+.^|`-d, --dry-run`
+|Instead of performing the rollback, describe what the rollback will look like in human-readable form.
 
-To rollback to a previous deployment:
+.^|`-o, --output`
+|Instead of performing the rollback, print the updated deployment configuration in the specified format: `json`\|`yaml`\|`template`\|`templatefile`.
+
+.^|`-t, --template`
+|Template string or path to template file to use when `-o=template` or `-o=templatefile`.
+|===
+
+To perform a rollback:
+
+====
 
 ----
-$ oc rollback <deployment>
+$ oc rollback deployment-1
```

Comment:
All the other examples use a <resource_description> style... why the change here?